### PR TITLE
Bug 1973525: [on-prem] Drop runlevel from infra namespaces

### PR DESCRIPTION
**Pull request by cybertron**
This appears to have been copy-pasta from the MCO namespace, but we
don't need it for the on-prem services.

**- What I did**

**- How to verify it**

**- Description for the changelog**
Dropped the runlevel specifier for the on-prem namespaces. It was not necessary and may have caused confusion about the services running in the namespace.

### CODE REVIEW

**Review by kikisdeliveryservice — APPROVED** [review]
Adding approve and hold remove hold once you get the lgtms you need =)

/hold

**Review by mandre — COMMENTED** [review]
/test e2e-openstack
LGTM provided e2e-openstack comes back green.

**Comment by yuvalk** [comment]
my PR is probably a dup
https://github.com/openshift/machine-config-operator/pull/2655
but
I think you've missed the run-level 1 for openshift-machine-config-operator

**Comment by cybertron** [comment]
> my PR is probably a dup
> #2655
> but
> I think you've missed the run-level 1 for openshift-machine-config-operator

Possibly. This change came out of a conversation about the infra namespaces, which clearly did not need the runtime attribute. I'm not certain whether that's the case for MCO itself.

**Comment by kikisdeliveryservice** [comment]
> > my PR is probably a dup
> > #2655
> > but
> > I think you've missed the run-level 1 for openshift-machine-config-operator
> 
> Possibly. This change came out of a conversation about the infra namespaces, which clearly did not need the runtime attribute. I'm not certain whether that's the case for MCO itself.

Just to followup, I closed the other PR in favor for this one that only touches on-prem things.

**Comment by cybertron** [comment]
/hold cancel

All affected platforms are passing and we have signoff from 3 of them. Let's get this in.

**Pull request merged**
